packages/accordions/src/elements/accordion/components/Section.tsx

```diff
@@ -9,6 +9,9 @@ import React, { forwardRef, useRef, HTMLAttributes } from 'react';
 import { useAccordionContext, SectionContext } from '../../../utils';
 import { StyledSection } from '../../../styled';
 
+/**
+ * @extends HTMLDivElement
```

Comment:
Should be `HTMLAttributes<HTMLDivElement>`

Reply:
Fixed.

Reply:
extends have been removed for now.